.pre-commit-hooks.yaml

```diff
@@ -1,16 +1,4 @@
 - id: prettier
   name: prettier
-  entry: prettier --write --list-different
+  entry: prettier --write --list-different --ignore-unknown
```

Comment:
@thorn0 Do you know how to test this?

Reply:
I don't use pre-commit, but I agree that it seems to be safe to remove.

Reply:
Hi @fisker ,
Yes, this is safe to change.

You can test on an empty repo, add a `.pre-commit-hooks.yaml` file with:
```yaml
repos:
  - repo: https://github.com/fisker/prettier
    rev: ignore-unknown
    hooks:
      - id: prettier
```

then 

```bash
pre-commit install
git add -A
pre-commit run -a
```

Reply:
Thank you  !